user_manual/android/android_app.rst

```diff
@@ -0,0 +1,115 @@
+==============================
+Using the ownCloud Android App
+==============================
+
+Accessing your files on your ownCloud server via the Web interface is easy and 
+convenient, as you can use any Web browser on any operating system without 
+installing special client software. However, the ownCloud Android app offers 
+some advantages over the Web interface:
+
+* A simplified interface that fits nicely on a tablet or smartphone
+* Automatic synchronization of your files
+* Instant uploads of photos or videos recorded on your Android device
+* Easily add files from your device to ownCloud
+* Two-factor authentication
+
+Getting the ownCloud Android App
+--------------------------------
+
+One way to get your ownCloud Android app is to log into your ownCloud server 
+from your Android device using a Web browser such as Chrome, Firefox, or 
+Dolphin. The first time you log in to a new ownCloud account you'll see a screen 
+with a download link to the ownCloud app in the `Google Play store
+<https://play.google.com/store/apps/details?id=com.owncloud.android>`_.
+
+.. figure:: /images/android-first-screen.jpg
+
+You will also find these links on your Personal page in the Web interface,
+
+You can also get it from the `Amazon App store 
+<http://www.amazon.com/ownCloud-Inc/dp/B00944PQMK/>`_, and get source code and 
+more information from the `ownCloud download page 
+<http://owncloud.org/install/#desktop>`_.
+
+Connecting to Your ownCloud Server
+----------------------------------
+
+The first time you run your ownCloud Android app it opens to a configuration 
+screen. Enter your server URL, login name, password, and click the Connect 
+button. (Click the eyeball to the right of your password to expose your 
+password.)
+
+.. figure:: /images/android-new-account.png
```

Comment:
The images are not being shown.
It seems to be because "images" folder, where they are stored, is not at the same level of this android_app.rst file.